lib/gollum-lib/wiki.rb

```diff
@@ -436,7 +421,7 @@ def update_page(page, name, format, data, commit = {})
       dir      = ::File.dirname(page.path)
       dir      = '' if dir == '.'
       filename = (rename = page.name != name) ?
-          Gollum::Page.cname(name) : page.filename_stripped
+          name : page.filename_stripped
```

Comment:
This is inconsistent at present: I switched to using `canonicalize_filename` later during constructing this PR. However, as I mentioned, the result is at present the same. So we could remove the `canonicalize_filename` method and just use `filename_stripped` where necessary.